doc/homepage.ipynb

```diff
@@ -13,6 +13,8 @@
     "* [Dask](http://dask.pydata.org): DataFrame, Series (distributed/out of core arrays and columnar data)\n",
     "* [Streamz](http://streamz.readthedocs.io): DataFrame(s), Series(s) (streaming columnar data)\n",
     "* [Intake](http://github.com/ContinuumIO/intake): DataSource (data catalogues)\n",
+    "* [GeoPandas](http://pandas.pydata.org): DataFrame, Series (columnar/tabular data)\n",
+    "\n",
     "\n",
     "Many of these libraries have the concept of a high-level plotting API that lets a user generate common plot types very easily. The native plotting APIs are generally built on [Matplotlib](http://matplotlib.org), which provides a solid foundation, but means that users miss out the benefits of modern, interactive plotting libraries for the web like [Bokeh](http://bokeh.pydata.org) and [HoloViews](http://holoviews.org).\n",
```

Comment:
Is "Several" or "Some" more accurate than "Many" now, given how long the list is?  Seems like Pandas and XArray do have these APIs already; are there others?  I don't think it's a majority...

Reply:
Sure, xarray, pandas and geopandas do.

Reply:
Ok, sounds like "Several" to me.